packages/pong-engine/src/queue.dto.ts

```diff
@@ -41,6 +47,18 @@ type GameFinishedDto = {
   gameRoomId?: never;
 };
 
+export enum gameQueueServerToClientWsEvents {
+  gameStatusUpdate = 'gameStatusUpdate',
+  gameChallenge = 'gameChallenge',
+}
+
+export enum gameQueueClientToServerWsEvents {
+  gameQueueJoin = 'gameQueueJoin',
+  gameQuitWaiting = 'gameQuitWaiting',
+  gameChallengeResponse = 'gameChallengeResponse',
+  gameUserChallenge = 'gameUserChallenge',
+}
```

Comment:
I think the Enums provide better code completion features, and they make the intent of the programmer more clear, AFAIK, since when we use them the type is embedded in the name: pretty much we're accesing a member of an object to use them.

As an alternative choice that would provide the same features I look for, I can mention how OpenApi generator uses actual exported objects to recreate the backend enums in webapp. But since I'm doing this by hand (and as long as I have to do this by hand), I don't see a plus using the object rather than the enum.

String types don't make it as readable, IMHO. I will not make this change.